portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,72 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300&family=Open+Sans&display=swap');
+
+background-color: #F1F5F4;
+  margin: 0px;
```

Comment:
use consistent indentation.

Reply:
This is strange, in my cloud shell editor, they seem to be evenly indented.

Reply:
Ah! It could be a `tab` and `spaces` thing! Check if the file has a mixed `spaces` and `tabs` usage. Chances are, the tab width is same as the number of spaces in use on the cloud shell but different on GitHub.